apis/apiextensions/v1/composition_transforms.go

```diff
@@ -481,17 +483,19 @@ func (c ConvertTransformFormat) IsValid() bool {
 // A ConvertTransform converts the input into a new object whose type is supplied.
 type ConvertTransform struct {
 	// ToType is the type of the output of this transform.
-	// +kubebuilder:validation:Enum=string;int;int64;bool;float64
+	// +kubebuilder:validation:Enum=string;int;int64;bool;float64;object;list
```

Comment:
@turkenh   should this be `array` instead of `list` ?  the docs PR captures it as `array`.

A Composition with `transforms[].convert.toType` set to array results in error:
```
the Composition "compose-v1alpha1.example.com" is invalid: 
* spec.resources[1].patches[3].transforms[0].convert.toType: Unsupported value: "array": supported values: "string", "int", "int64", "bool", "float64", "object", "list"
```

trying to apply with `list` also results in error:
```
The Composition "compose-v1alpha1.example.com" is invalid: 
* spec.resources[1].patches[3].transforms[0].convert.toType: Invalid value: "list": invalid type
```